pkg/util/validation/limits.go

```diff
@@ -193,6 +196,8 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.MaxGlobalSeriesPerUser, MaxSeriesPerUserFlag, 150000, "The maximum number of in-memory series per tenant, across the cluster before replication. 0 to disable.")
 	f.IntVar(&l.MaxGlobalSeriesPerMetric, MaxSeriesPerMetricFlag, 0, "The maximum number of in-memory series per metric name, across the cluster before replication. 0 to disable.")
 
+	f.IntVar(&l.MaxGlobalEphemeralSeriesPerUser, MaxEphemeralSeriesPerUserFlag, 150000, "The maximum number of in-memory ephemeral series per tenant, across the cluster before replication. 0 to disable.")
```

Comment:
I think it would be better to rename the variable `MaxEphemeralSeriesPerUserFlag` to `MaxGlobalEphemeralSeriesPerUserFlag` so that it matches the name of the setting. Unfortunately, the line above has the same inconsistency where `Global` is skipped, so I guess consistent inconsistency wins

Reply:
I think we can drop `global` from the new parameter. We used it before because we had both "local" and "global" limits, but we no longer have "local" limits, so we don't need to use this naming for new limits.

Reply:
Done in https://github.com/grafana/mimir/pull/4004/commits/6b2839d148d6267f412e6e89524a3ec788dbad08.

Reply:
thanks!